src/ec/packages/ec-component-language-list/language-list-overlay.html.twig

```diff
@@ -80,31 +80,12 @@
         {{- _title -}}
       </div>
     </div>
-    <div class="ecl-row">
```

Comment:
Please also remove columns' calculations from this template https://github.com/ec-europa/ecl-twig/pull/86/files#diff-bdc8318b34f6b77a26540827fc65fbf5R47

They are handled now by the `language-list`.